server/pbench/bin/pbench-base.sh

```diff
@@ -78,7 +78,7 @@ export mail_recipients=$(getconf.py mailto pbench-server)
 # make all the state directories for the pipeline and any others needed
 LINKDIRS="TODO TO-UNPACK TO-COPY-SOS TO-SYNC SYNCED TO-LINK TO-INDEX TO-BACKUP \
 	INDEXED WONT-INDEX DONE BAD-MD5 SATELLITE-MD5-PASSED SATELLITE-MD5-FAILED \
-	TO-DELETE SATELLITE-DONE TO-UNPACK UNPACKED MOVED-UNPACKED"
+	TO-DELETE SATELLITE-DONE TO-UNPACK UNPACKED MOVED-UNPACKED COPIED-SOS"
```

Comment:
Can you reorder these, so that they are paired properly? Just  put the related states together (e.g. TO-INDEX INDEXED WONT-INDEX). Put any general final states (e.g. BAD_MD5) at the end on their own line and add a comment to indicate that they are final states.